app/src/main/java/com/example/searchmovies/domain/dto/Movie.kt

```diff
@@ -2,5 +2,11 @@ package com.example.searchmovies.domain.dto
 
 data class Movie(
     val title: String,
-    val releaseDate: String
-)
\ No newline at end of file
+    val releaseDate: String,
+    val popularity: Double,
+    val voteAverage: Double,
+    val voteCount: Int,
+    val adult: Boolean,
```

Comment:
W przypadku typu Boolean, warto używać przedrostka "is", czyli w tym przypadku "isAdult".